apps/mull-ui/src/app/components/event-page-info/event-page-info.tsx

```diff
@@ -52,11 +52,11 @@ export const EventPageInfo = ({
       </div>
       <div className="info-row">
         <FontAwesomeIcon icon={faMapMarkerAlt} className="event-page-icon color-grey" />
-        {/* TODO: Remove placeholder text once location is implemented */}
         <p className="row-text" data-testid="event-location">
-          18 Mull Ave, Pointe-Claire, Montreal
+          {/* TODO: placeholder till TASK-33 is done uncomment the line below to see on preview page*/}
+          {/* {event.location.title} */}
+          Test Location
```

Comment:
If I recall correctly, I fixed this already, I believe I fixed this already when I was with @JimLau49 and @ys-lin. It seems to have been reverted :(